src/Background/estimators.jl

```diff
@@ -61,3 +61,30 @@ julia> estimate_background(Mode, data)
 struct Mode <: BackgroundEstimator end
 
 estimate_background(::Mode, data; dims = :) = mode(data)
+
+
+"""
+    MMMBackground(median_factor=3, mean_factor=2)
+
+This estimator is originally taken from DAOPHOT.
+The background is estimated as `median_factor*median - mean_factor*mean`, the default values of `median_factor` and `mean_factor` are `3` anf `2` respectively.
+
+# Example
+```jldoctest
+julia> x = ones(5,5);
+
+julia> estimate_background(MMMBackground(), x)
+1.0
+
+julia> estimate_background(MMMBackground(), x, dims = 1)
+1×5 Array{Float64,2}:
+ 1.0  1.0  1.0  1.0  1.0
```

Comment:
I think a nice example of the differences in this constructor would be:
```julia
julia> estimate_background(MMMBackground, x)
1.0

julia> estimate_background(MMMBackground(4, 3), x, dims=1)
5×5 Array{Float64,2}:
 1.0  1.0  1.0  1.0  1.0
```

Reply:
I’d still like to see this change so the users’ example shows both methods of construction.

Reply:
I have added the suggested examples, at lines 113-119 in src/Background/estimators.jl